android/app/build.gradle

```diff
@@ -65,6 +65,10 @@ android {
            signingConfig signingConfigs.release
        }
    }
+
+   configurations {
+    implementation.exclude module:'protobuf-lite' 
+   }
```

Comment:
This should not be here - maybe not needed anymore now that protobuf changes have been reverted